gulpfile.babel.js

```diff
@@ -1,18 +1,36 @@
 import gulp from 'gulp';
 import requireDir from 'require-dir';
-import livereload from 'gulp-livereload';
+import browserSync from 'browser-sync';
 
 requireDir( './gulp-tasks' );
 
+const bs = browserSync.create();
+
+gulp.task( 'bs-reload-css', ( cb ) => {
+	bs.reload('*.css');
+	cb();
+});
+
+gulp.task( 'bs-reload', ( cb ) => {
+	bs.reload();
+	cb();
+});
+
 gulp.task( 'js', gulp.series( 'webpack' ) );
 
 gulp.task( 'cssprocess', gulp.series( 'css', 'cssnano', 'cssclean' ) );
 
 gulp.task( 'watch', () => {
 	process.env.NODE_ENV = 'development';
-	livereload.listen( { basePath: 'dist' } );
-	gulp.watch( ['./assets/css/**/*.css', '!./assets/css/src/**/*.css'], gulp.series( 'cssprocess' ) );
-	gulp.watch( './assets/js/**/*.js', gulp.series( 'js' ) );
+	bs.init({
+		proxy: 'local-theme-url.test',
```

Comment:
there's a search and replace value for URL in the generator, it's: tenup-scaffold.test - however, I think we need to find a way to bubble this up better rather than burying it in a gulp file. Part of the goal in creating the scaffolding is that it should work out of the box. I'd like to stay away from starting a pattern where we need to manually update lines of code before starting work.

Reply:
Would you be more comfortable having either something like a `config.json` file, or maybe having the local URL in the `package.json` file?

Reply:
I would vote for a `config.json` and even perhaps an optional `config-local.json` for Engineers to override certain things on their local's.